src/Resources/ServiceOptionPrice.php

```diff
@@ -106,6 +107,26 @@ public function getCurrency()
             : null;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function setRequired($required)
+    {
+        $this->attributes[self::ATTRIBUTE_REQUIRED] = $required;
+
+        return $this;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isRequired()
+    {
+        return isset($this->attributes[self::ATTRIBUTE_REQUIRED])
+            ? $this->attributes[self::ATTRIBUTE_REQUIRED]
+            : null;
```

Comment:
According to the interface docblock, `isRequired()` can only return a `boolean`. Maybe add `null` to the docblock too, if `null` is an option?